plugin/pkg/admission/limitranger/admission.go

```diff
@@ -421,13 +421,6 @@ func (d *DefaultLimitRangerActions) SupportsAttributes(a admission.Attributes) b
 		return false
 	}
 
-	// Since containers and initContainers cannot currently be added, removed, or updated, it is unnecessary
-	// to mutate and validate limitrange on pod updates. Trying to mutate containers or initContainers on a pod
-	// update request will always fail pod validation because those fields are immutable once the object is created.
-	if a.GetKind().GroupKind() == api.Kind("Pod") && a.GetOperation() == admission.Update {
```

Comment:
```go
if a.GetKind().GroupKind() == api.Kind("Pod") && a.GetOperation() == admission.Update && !utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
	return false
}
```

Would it be better to still retain this part of the decision logic?

Reply:
The part of the decision logic is to fix the bug mentioned in #62666 . Does the bug still exist ? 



 The featuregate condition judge added just  make it more confused to others . Can we find a better way to fix ?

@togettoyou

Reply:
adding a FeatureGate condition is to avoid unnecessary processing when the InPlacePodVerticalScaling feature is not enabled. As for whether the bug will recur, more test cases need to be written.

Reply:
ok